metrics/bucket.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// NewInt64ByteBucket creates a Int64Bucket with units of bytes and a default set of aggregation
+// bounds from .25 kilobytes to 10 megabytes.
+func NewInt64ByteBucket(name, desc string, tagKeys ...tag.Key) *Int64Bucket {
```

Comment:
"Bucket" is not a term widely used for metrics AFAIK. Use of the verb `Add` as a method below suggests that  that this is functioning as a *counter*, counting all the bytes. An alternative we could use is a *histogram* if we wanted to understand the distribution of values (but then the method should be `Record` or similar). Please use one of those words.

We already have an `Int64Counter`, so if that's the intention we might just need some more construction methods that supply different units and bounds. We don't already have `Int64Histogram`, but this could be it.